chrome/content/zotero/zoteroPane.xul

```diff
@@ -245,14 +258,14 @@
 					<menuitem class="menuitem-iconic zotero-menuitem-new-collection" label="&zotero.toolbar.newCollection.label;" command="cmd_zotero_newCollection"/>
 					<menuitem class="menuitem-iconic zotero-menuitem-new-saved-search" label="&zotero.toolbar.newSavedSearch.label;" command="cmd_zotero_newSavedSearch"/>
 					<menuitem class="menuitem-iconic zotero-menuitem-new-collection" label="&zotero.toolbar.newSubcollection.label;" oncommand="ZoteroPane_Local.newCollection(ZoteroPane_Local.getSelectedCollection().key)"/>
-					<menuitem class="menuitem-iconic zotero-menuitem-new-feed" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
-					<menuitem class="menuitem-iconic zotero-menuitem-refresh-feed" label="&zotero.toolbar.refreshFeed.label;" oncommand="ZoteroPane_Local.refreshFeed();"/>
+					<menuitem class="menuitem-iconic zotero-menuitem-new-feed" label="&zotero.toolbar.feeds.new;" command="cmd_zotero_newFeed_fromURL"/>
```

Comment:
This menuitem isn't actually used, is it? I don't think it appears anywhere (because the Feeds header isn't selectable), and the `newFeed` position isn't referenced in buildCollectionContextMenu.